app/services/support_interface/add_course_choice_after_submission.rb

```diff
@@ -0,0 +1,23 @@
+module SupportInterface
+  class AddCourseChoiceAfterSubmission
+    attr_reader :application_form, :course_option, :send_to_provider
+
+    def initialize(application_form:, course_option:, send_to_provider: true)
+      @application_form = application_form
+      @course_option = course_option
+      @send_to_provider = send_to_provider
+    end
+
+    def call
+      application_choice = ApplicationChoice.create!(
+        application_form: application_form,
+        course_option: course_option,
+        status: 'awaiting_provider_decision',
+      )
+
+      SendApplicationToProvider.call(application_choice) if send_to_provider
```

Comment:
I think this shouldn't be optional:

```suggestion
      SendApplicationToProvider.call(application_choice)
```